docs/content/howto/logging/send_table.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Send tables to Rerun
+order: 0
+description: Shows how to send tables as dataframes to the Rerun viewer.
+---
+
+> **Note:** The `send_table` API is currently experimental and may change in future releases.
+
+Rerun now supports sending tabular data to the Rerun Viewer! This feature allows you to visualize and interact with dataframes (encoded as Arrow record batches) directly in the Rerun Viewer environment.
+
+## Overview
+
+The `send_table` API provides a straightforward way to send tabular data to the Rerun Viewer. This is particularly useful for:
+
+- Inspecting dataframes alongside other visualizations
+- Debugging data processing pipelines
+- Presenting structured data in a readable format
+
+## References
+
+For complete examples of using `send_table`, please refer to:
+
+- [🐍 Jupyter Notebook](https://github.com/rerun-io/rerun/blob/main/examples/python/notebook/send_table.ipynb)
+- [🐍 Python SDK](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/howto/send_table.py)
+
+## Prerequisites
+
+- Rerun SDK (Python)
+- PyArrow library
+- Pandas
+- NumPy
+
+Which can be installed via:
+
+```sh
+pip install rerun-sdk[notebook] pyarrow pandas numpy
+```
+
+## Basic usage
+
+### Connecting to the Viewer
+
+```python
+from rerun_bindings import ViewerClient
+
+# Connect to a running Rerun Viewer
+client = ViewerClient(addr="rerun+http://0.0.0.0:9876")
```

Comment:
Is there no specific `/endpoint` for this?

Reply:
Yes, it uses `/proxy` (our default endpoint), I will add it for consistency.